gitpod-io/gitpod
# Update gpctl go modules

Signed-off-by: ArthurSens <arthursens2005@gmail.com>

## Description

Add missing modules so we can compile `gpctl` again

## Related Issue(s)

Fixes #5786

## How to test

Open a workspace in this PR and run:
```
cd dev/gpctl
go install
```

There should be no errors 

## Release Notes

```release-note
NONE
```
